package/gluon-web-private-wifi/i18n/de.po

```diff
@@ -13,18 +13,39 @@ msgstr ""
 msgid "8-63 characters"
 msgstr "8-63 Zeichen"
 
+msgid "Disabled"
+msgstr "Deaktiviert"
+
 msgid "Enabled"
 msgstr "Aktiviert"
 
+msgid "Encryption"
+msgstr ""
```

Comment:
Translations are missing here and below.

Reply:
Good catch. This is however something for the private-wpa3 PR. Will fix it over there.